.gitignore

```
@@ -9,6 +9,7 @@
 
 # dependencies
 /node_modules
+/**/node_modules
```

```suggestion
node_modules
```

alone should cover both cases

---

Done.